### PR TITLE
cf: add page

### PR DESCRIPTION
**Pull request by waciumawanjohi**
Authored-by: Waciuma Wanjohi <code@waciuma.com>

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/cf.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# cf
+
+> Command line tool to manage apps and services on Cloud Foundry.
+> More information: <https://docs.cloudfoundry.org/>.
+
+- Push an app the easiest way, using default settings:
```
Comment (owenvoke) [inline pages/common/cf.md:6]: This can probably be simplified to:
```suggestion
- Push an app using the default settings:
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @waciumawanjohi, thanks for the page, it looks great!

Just one minor suggestion to simplify the first example. 👍🏻

**Review by sbrl — APPROVED** [review]
Looks ok after @owenvoke's comments are addressed :-)

**Comment by waciumawanjohi** [comment]
Updated the page. 🙂

**Review by owenvoke — APPROVED** [review]
LGTM!

File: pages/common/cf.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# cf
+
+> Command line tool to manage apps and services on Cloud Foundry.
+> More information: <https://docs.cloudfoundry.org/>.
```
Comment (navarroaxel) [inline pages/common/cf.md:4]:
```suggestion
> More information: <https://docs.cloudfoundry.org>.
```

File: pages/common/cf.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`cf push {{app_name}}`
+
+- View the services available from your organization:
+
+`cf marketplace`
+
+- Create a service instance:
+
+`cf create-service {{service}} {{plan}} {{chosen_service_name}}`
```
Comment (navarroaxel) [inline pages/common/cf.md:16]:
```suggestion
`cf create-service {{service}} {{plan}} {{service_name}}`
```

File: pages/common/cf.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Connect an application to a service:
+
+`cf bind-service {{app_name}} {{chosen_service_name}}`
+
+- Run a script whose code is included in the app, but runs independently:
+
+`cf run-task {{app_name}} {{script_command}} --name {{task_name}}`
```
Comment (navarroaxel) [inline pages/common/cf.md:24]: Could we enclose {{script_command}} in quotes please?

```suggestion
`cf run-task {{app_name}} "{{script_command}}" --name {{task_name}}`
```

Passing code as a CLI argument may potentially include special characters and more than 1 word, which we want to ensure get passed correctly.

File: pages/common/cf.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`cf marketplace`
+
+- Create a service instance:
+
+`cf create-service {{service}} {{plan}} {{chosen_service_name}}`
+
+- Connect an application to a service:
+
+`cf bind-service {{app_name}} {{chosen_service_name}}`
```
Comment (navarroaxel) [inline pages/common/cf.md:20]: The service is named by the user, `service_name` is not enough clear about that?
```suggestion
`cf bind-service {{app_name}} {{service_name}}`
```

**Pull request merged**